### PR TITLE
update documentation and contribution guidelines to reflect SignalFx

### PR DESCRIPTION
**Pull request by charliesignalfx**

### CODE REVIEW

File: plugins/outputs/signalfx/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+```toml
+    ## SignalFx API Token
+    APIToken = "my-secret-key" # required.
+
+    ## Ingest URL
+    DatapointIngestURL = "https://ingest.signalfx.com/v2/datapoint"
+    EventIngestURL = "https://ingest.signalfx.com/v2/event"
+    
+    ## Exclude metrics by metric name
+    Exclude = ["plugin.metric_name", ""]
```
Comment (keitwb) [inline plugins/outputs/signalfx/README.md]: What's up with the blank string as the second item?

File: README.md [hunk 2]
```diff
@@ -241,6 +166,7 @@ formats may be used with input plugins supporting the `data_format` option:
 * [file](./plugins/outputs/file)
 * [graphite](./plugins/outputs/graphite)
 * [graylog](./plugins/outputs/graylog)
+* [influxdb](./plugins/outputs/influxdb)
```
Comment (keitwb) [inline README.md]: Way to knock them down.

File: plugins/outputs/signalfx/README.md (hunk 1 above)
Comment (charliesignalfx) [reply plugins/outputs/signalfx/README.md]: will address

**Comment by molner** [comment]
LGTM but I don't have write access on the repo... @charliesignalfx can you provide?

**Comment by molner** [comment]
LGTM

**Pull request merged**